src/frequenz/sdk/microgrid/_battery/_status.py

```diff
@@ -45,10 +40,25 @@ class BatteryStatus(Enum):
     """Component is working"""
 
 
+@dataclass
+class RequestResult:
```

Comment:
Does this have to go to `microgrid/_battery`?  I think this is specific to the power distributor.

Reply:
can we also rename this to `SetPowerResult` or something?  `RequestResult` sounds too confusing.

Reply:
I moved whole file to frequez.sdk.actor.power_distributing.
Renamed RequestResult to SetPowerResult.